client/my-sites/post-share/connection.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from 'components/forms/form-toggle/compact';
+import classNames from 'classnames';
+import SocialLogo from 'social-logos';
+
+const PostShareConnection = ( {
+	connection,
+	isActive,
+	onToggle,
+} ) => {
+	const {
+		external_display,
```

Comment:
Actually @jhnstn, `external_connection` is a property of the `connection` object. We are destructing it. Rename this field, as well as `keyring_connection_ID`, will demand an effort that maybe won't make sense since this component will be refactored soon (I hope).
It isn't the primary purpose of this PR make changes so profound.